README.md

```diff
@@ -439,3 +439,30 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
 
 
 </details>
+
+<details>
+  <summary>milkomeda</summary>
+
+  This endpoint is used to get milkomeda info data (minValue, address & tokens list per deployment/network)
+
+  Input
+
+  None (GET request)
+
+  Output
+
+  ```js
+  {
+    minimumValue: number,
+    address: string,
+    tokensPerNetwork: { // proposed networks are the most probable for now
+      internalTestnet: {
+        tokens: [ { name: string, policyId: string }, ]
```

Comment:
This list could end up being big in the future, so probably best not to send all the networks together in one endpoint. The backend deployment is network-specific anyway so we would only ever look at one of the values in the response

Reply:
Good point. I wasn't sure how would you like to have payload exposed (that's why there's only one endpoint). I'll add another one, so that it is possible to get tokens per each network.